packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -0,0 +1,60 @@
+Name:     foreman-release
+Version:  1.19.0
+Release:  1%{?dist}
+
+Summary:  Foreman repositories meta-package
+Group:    Applications/System
+License:  GPLv3+
+URL:      http://theforeman.org
+Source0:  %{name}.repo
```

Comment:
You can't use `%{name}` macro anymore since that's now `foreman-release` instead of `foreman`.